queries/blade/injections.scm

```diff
@@ -0,0 +1,16 @@
+((text) @injection.content
+  (#not-has-ancestor? @injection.content "envoy")
```

Comment:
Those are relatively expensive. It may be better to default to the more common one and override it in a second stanza.

Reply:
so you mean like this?

```scheme
((text) @injection.content
  (#set! injection.combined)
  (#set! injection.language php))

((text) @injection.content
  (#has-ancestor? @injection.content "envoy")
  (#set! injection.combined)
  (#set! injection.language bash))
```

Reply:
Yes.